assets/javascripts/discourse/components/query-result.js.es6

```diff
@@ -211,13 +267,23 @@ const QueryResultComponent = Ember.Component.extend({
     });
   },
 
+  _cutGraphLabel(label) {
```

Comment:
minor, I think I would go with ellipsis in the name instead of cut

Reply:
Good idea.